test/smoke/src/main.ts

```diff
@@ -310,30 +310,27 @@ describe(`VSCode Smoke Tests (${opts.web ? 'Web' : 'Electron'})`, () => {
 	// 		setupDataMigrationTests(opts['stable-build'], testDataPath);
 	// 	});
 	// }
+	before(async function () {
+		const app = new Application(this.defaultOptions);
+		await app!.start(opts.web ? false : undefined);
+		this.app = app;
+	});
 
-	describe(`VSCode Smoke Tests (${opts.web ? 'Web' : 'Electron'})`, () => {
```

Comment:
Actually could you just comment out the nested suite instead and add SQL CARBON EDITs? Makes it easier to do merges later on